docs/en/serverless/quickstarts/auto-detect-logs-metrics.mdx

```diff
@@ -0,0 +1,96 @@
+---
+slug: /serverless/observability/quickstarts/auto-detect-logs-metrics
+title: Auto-detect logs and metrics
+description: Learn how to scan your hosts to detect and collect logs and metrics.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+
+In this quickstart guide, you'll learn how to scan your host to detect and collect logs and metrics,
+then navigate to dashboards to further analyze and explore your observability data.
+You'll also learn how to get value out of your observability data.
+
+## Prerequisites and limitations
+
+This quickstart has the following requirements and limitations:
+
+- The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to <DocLink slug="/serverless/general/assign-user-roles" />.
+- Root privileges on the host are required to run the auto-detection script used in this quickstart.
+- The auto-detection script currently scans for metrics and logs from Apache, Docker, Nginx, and the host system.
+   It also scans for custom log files.
+- The auto-detection script works on Linux and MacOS only and requires support for the `lsof` command.
+
+## Detect and collect observability data
+
+1. <DocLink slug="/serverless/observability/create-an-observability-project">Create a new ((observability)) project</DocLink>, or open an existing one.
+1. In your ((observability)) project, go to **Add Data**.
+1. Select **Collect and analyze logs**, and then select **Auto-detect logs and metrics**.
+1. Copy the command that's shown. For example:
+  ![Quick start showing command for running auto-detection](../images/quickstart-autodetection-command.png)
+  You'll run this command to download the auto-detection script and scan your system for observability data.
+1. Open a terminal on the host you want to scan, and run the command.
+1. Review the list of log files:
+    - Enter `Y` to ingest all the log files listed
+    - Enter `n` to either exclude log files or specify additional log paths. Enter `Y` to confirm your selections.
+
+  The script downloads and installs the software needed to collect observability data from the host and send it to Elastic.
```

Comment:
Suggestion: "The script downloads and installs Elastic Agent which is used to collect observability data..."

We only install Elastic Agent, keeping the statement too broad might worry users what software we're going to install